app/models/book.rb

```diff
@@ -9,4 +16,64 @@ class Book < ApplicationRecord
   has_many :borrow_requests, through: :borrow_request_items
   has_many :reviews, dependent: :destroy
   has_many :favorites, as: :favorable, dependent: :destroy
+
+  validates :title,
+            presence: {message: I18n.t("activerecord.errors.models.book.attributes.title.blank")},
+            length: {
+              maximum: MAX_TITLE_LENGTH,
+              message: I18n.t("activerecord.errors.models.book.attributes.title.too_long", count: MAX_TITLE_LENGTH)
```

Comment:
```suggestion
  validates :title,
            presence: true,
            length: {maximum: MAX_TITLE_LENGTH)
```
Các validates này có key mặc định rồi, fix tương tự những chỗ khác